elastic/eui
# [EuiTabs] Fix link tabs incorrectly wrapping

## Summary

🤦 I fixed this for `<button>` tab content in #7309 but missed link tabs. Originally broken in #6311

### Before:

<img src="https://raw.githubusercontent.com/elastic/eui/31f57db8eeb495d00852890bbf85014368c32c8c/packages/eui/.loki/reference/chrome_desktop_Navigation_EuiTabs_EuiTabs_Long_Tab_Content.png" alt="">

### After

<img src="https://raw.githubusercontent.com/elastic/eui/885bdb59291882f25c889b67ac69266851e0915d/packages/eui/.loki/reference/chrome_desktop_Navigation_EuiTabs_EuiTabs_Long_Tab_Content.png" alt="">

## QA

- Go to https://eui.elastic.co/pr_8073/#/navigation/tabs
- Open devtools and responsive/mobile mode, change screen size to 400px~
- [x] Confirm the first `Monosodium Glutamate` tab example on the page does not wrap incorrectly

### General checklist

- Browser QA
    ~- [ ] Checked for **accessibility** including keyboard-only and screenreader modes~
    ~- [ ] Checked in both **light and dark** modes~
- Docs site QA - N/A, style bugfix
- Code quality checklist
    ~- [ ] Added or updated **[jest](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/unit-testing.md) and [cypress](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/cypress-testing.md) tests**~
- Release checklist
    ~- [ ] If applicable, added the **breaking change** issue label (and filled out the breaking change checklist)~
- Designer checklist - N/A
